patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -723,12 +723,41 @@
 +    {
 +        return null;
 +    }
++    
++    /**
++     * Vanilla's default glint value
++     * */
++    @SideOnly(Side.CLIENT)
++    private static final int defaultGlint = -8372020;
```

Comment:
Might be better to put this constant in hexadecimal notation rather than int so you can actually get an understanding of what's going on.

Example: 0xFF8040CC